ui/admin/src/assets/i18n/en.json

```diff
@@ -159,19 +162,18 @@
   "Checkout duration": "Checkout duration",
   "Checkout duration is required.": "Checkout duration is required.",
   "Circulation Policy": "Circulation Policy",
-  "Number renewals": "Number renewals",
-  "Checkout duration must be great than 0.": "Checkout duration must be great than 0.",
+  "Number renewals": "Number of renewals",
```

Comment:
Should be changed in the original string?

Reply:
Ok